spec/sinatra_ar_crud_lab_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'spec_helper'
+
+describe "Blog Post App" do
+  let(:post_name) { "Hello World" }
+  let(:post_content) { "This is my first blog post!!!" }
+
+  before do
+    @post1 = Post.create(:name => post_name, :content => post_content)
+    @post2 = Post.create(:name => "second post", :content => "i'm a really good blogger")
+  end
+
+  describe "Create Action" do 
+
+    it "creates a new blog post" do
+      visit '/posts/new'
+
+      fill_in :name, :with => "my favorite blog post"
+      fill_in :content, :with => "blogging!!!!"
+
+      click_button 'submit'
+
+      expect(Post.all.count).to eq(3)
+      expect(Post.last.name).to eq("my favorite blog post")
+    end
+
+  end
+
+  describe "Read Action " do
+
+    it 'index action responds with a 200 status code' do
+      get "/posts"
+      expect(last_response.status).to eq(200)
+    end
+
+    it "index action displays all the blog posts" do
+      get "/posts"
+      expect(last_response.body).to include(post_name)
+      expect(last_response.body).to include(@post2.name)
+    end
+
+    it 'show page responds with a 200 status code' do
+      get "/posts/#{@post1.id}"
+      expect(last_response.status).to eq(200)
+    end
+
+    it "show page displays the post's name" do
+      get "/posts/#{@post1.id}"
+      expect(last_response.body).to include(post_name)
+    end
+
+    it "show page displays the post's content" do
+      get "/posts/#{@post1.id}"
+      expect(last_response.body).to include(post_content)
+    end
+
+   
+  end
+
+
+  describe "UPDATE ACTION" do
+
+
+    it 'responds with a 200 status code' do
+      get "/posts/#{@post2.id}/edit"
+      expect(last_response.status).to eq(200)
+    end
+
+    it "saves edits a blog post" do
```

Comment:
Do you want to also ensure that the fields are filled with the existing object? So it's not a bunch of blank fields?